cromshell

```diff
@@ -936,6 +1175,22 @@ if ${ISCALLEDBYUSER} ; then
   # Get our sub-command:
   SUB_COMMAND=${1}
   shift
+  
+  # Check if we need to set this up.
+  # Note: because of how `notify` works, we can't require the setup for the notify action.
+  if ${CROMWELL_NEEDS_SETUP} && [[ "${SUB_COMMAND}" != "notify" ]] ; then
```

Comment:
maybe try without this condition and see if it works?

Reply:
Just tested it.  It doesn't work because you're effectively calling it when you run the `ssh` command.